shepherd_bms/src/analyzer.cpp

```diff
@@ -59,6 +59,7 @@ void Analyzer::calcPackTemps()
     bmsdata->max_temp = {MIN_TEMP, 0, 0};
     bmsdata->min_temp = {MAX_TEMP, 0, 0};
     int total_temp = 0;
+    int total_seg_temp = 0;
     for(uint8_t c = 1; c < NUM_CHIPS; c++)
     {
         for(uint8_t therm = 17; therm < 28; therm++)
```

Comment:
I feel like we have macros for these somewhere, but if not all good. Try and avoid magic numbers wherever possible

Reply:
🤷‍♂️ not sure that was there before

Reply:
Yeah I don't recall proper macros for this being made tbh. There was an argument to not since y'all wanted to implement a more thorough disable unused therms function and then we would change all of these to just pull 0-10, since that's more intuitive. Given that doesn't sound like it'll happen though we could prob switch everything to macros